src/Components/Title.tsx

```diff
@@ -0,0 +1,14 @@
+export default function Title(props: { title: string }) {
```

Comment:
I like this tiny component (with typed props)! Some of the others should follow it's lead ;)